platform/authorized_api_guide.md

```diff
@@ -20,15 +21,57 @@ API credentials which then allows you to request for a token that is used for au
   Name must contains at least 3 characters.
 * Copy your newly generated `CLIENT ID` and `CLIENT SECRET` because they **won't be available for you after dismissing this window.**
 
-### Create access token
+#### Create access token
 
-Having generated your API Credentials, now you are ready for creating access token that will be used in communication with API.
-To do so, send POST request 
+Having generated your API Credentials, now you are ready for creating access token that will be used in 
+communication with API.
+
+Piwik PRO API tokens use [JWT](https://jwt.io/) format.
+
+Make POST call to `https://<domain>/auth/token` with header `Content-Type: application/json` and payload: 
+`{ grant_type: 'client credentials', client_id: '<clientId>', client_secret: '<clientSecret>' }`.
+
+Response example:
 ```
-POST /auth/token
+{"token_type":"Bearer","expires_in":86400,"access_token":"<your_access_token>"}
 ```
 
-Curl example:
+Now, you can use obtained `<access_token>` for communication with Piwik PRO API.
+Field `expires_in` stands for time (in seconds) for token expiration (TTL).
+As token is a Bearer type, it must be **included in every API call** within header.
+
+```
+Authorization: Bearer <your-token-here>
+```
+  
+### Deleting API Credentials
+
+Once you want to revoke the possibility of generating API token using given `CLIENT ID` and `CLIENT SECRET`,
+go to `My profile` and click `Delete` button on selected API credentials.
+
+## API usage example
+
+Whatever API call you choose, first remember that you must generate 
```

Comment:
I rather understand that this way: I have previously generated credentials and I HAD token generated week ago but after this time I want to perform some requests to API... so this section shouldn't include information about:
1. first - have generated new (or old but saved somewhere) credentials
2. second - generate a FRESH token (if expired but generating new one could be a preferable way) required for every single request
???

Reply:
as we talk offline I will try to be more precise with generation of token & api credentials